### ISSUE TITLE
Tobii Pro Vive SDK removing SteamVR SDK dependency

### ISSUE THREAD

**Issue by Naphier** [Tobii/UnitySDK#5]
Hello, I know this is not the Pro Vive SDK, but wanted to get in contact with the devs for that. The Pro Vive SDK relies on SteamVR package installed in Unity. However, I think this is now unnecessary. I don't see where SteamVR capabilities are used beyond getting the distance between the lenses and distance from head to lenses. This can be accomplished with UnityEngine.XR functionality just as easy (if not easier).

A couple very simple changes to you VRUtility class would remove the dependency on SteamVR.

To get distance from eye to head (using UnityEngine.XR):
```
var openVrHead2EyeDepth = Vector3.Distance(
                    InputTracking.GetLocalPosition(XRNode.CenterEye),
                    InputTracking.GetLocalPosition(XRNode.Head));
```

To get distance between eyes:
```
var openVrIpd = Vector3.Distance(
                        InputTracking.GetLocalPosition(XRNode.LeftEye), 
                        InputTracking.GetLocalPosition(XRNode.RightEye));
```

This would also reduce your reflection calls, but in all likelihood, Unity is also using reflection to get these values.

I'm not seeing anywhere else you guys are using SteamVR SDK. Can you please confirm that? Not having to implement SteamVR SDK in our project will reduce our integration efforts by about 50%. We don't need the full SteamVR SDK and from what I can tell Tobii doesn't either.

Cheers and happy new year!

**Comment by mattiasbrand** [comment]
Hi @Naphier! Are you referring to the Tobii XR SDK (https://developer.tobii.com/vr/develop/api-reference/sdk/)?

I'd be happy to have a more generic solution to the offset issue than using OpenVR. I'll try out your suggestion!

**Comment by Naphier** [comment]
Nope! Didn't even know that existed. We were pointed to this one: 
https://www.tobiipro.com/product-listing/tobii-pro-sdk/

What's the difference between the two? Can't easily find that information on the websites. :(
